src/frontend/apps/drive/src/features/explorer/components/grid/ExplorerGrid.tsx

```diff
@@ -163,170 +159,184 @@ export const ExplorerGrid = (props: ExplorerProps) => {
     }
 
     return (
-      <div className="c__datagrid__table__container">
-        <table ref={tableRef} tabIndex={0} onKeyDown={onKeyDown}>
-          <thead>
-            <tr>
-              {/* This one stands for the mobile column */}
-              <th></th>
-              <th style={{ width: "50%" }}>
-                <div className="c__datagrid__header fs-h5 c__datagrid__header--sortable">
-                  {t("explorer.grid.name")}
-                </div>
-              </th>
-              <th>
-                <div className="c__datagrid__header fs-h5 c__datagrid__header--sortable">
-                  {t("explorer.grid.last_update")}
-                </div>
-              </th>
-              <th></th>
-            </tr>
-          </thead>
-          <tbody>
-            {table.getRowModel().rows.map((row) => {
-              const isSelected = !!selectedItemsMap[row.original.id];
-              const isOvered = !!overedItemIds[row.original.id];
-              return (
-                <tr
-                  key={row.original.id}
-                  className={clsx("selectable", {
-                    selected: isSelected,
-                    over: isOvered,
-                  })}
-                  data-id={row.original.id}
-                  tabIndex={0}
-                  onClick={(e) => {
-                    const target = e.target as HTMLElement;
-                    const closest = target.closest("tr");
-                    // Because if we use modals or other components, even with a Portal, React triggers events on the original parent.
-                    // So we check that the clicked element is indeed an element of the table.
-                    if (!closest) {
-                      return;
-                    }
+      <ExplorerGridItemsList
+        items={props.childrenItems}
+        gridActionsCell={props.gridActionsCell}
+        selectedItemsMap={selectedItemsMap}
```

Comment:
having `selectedItemsMap` and `selectedItems` which stores the same state is redundant nope?

Reply:
I fixed it.